website/docs/appflow/web/getting-started.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Getting Started
+sidebar_label: Getting Started
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Getting Started
+
+Appflow is the tool used to build and distribute your web bundles across your team and deploy Live Updates to your end users. 
+
+You can login to Appflow [here](https://ionic.io/login), either using your email address or an org slug if one has been provided to you. 
+
+When going through this tutorial, it will be beneficial to keep in mind the structure of your organizaton and how the responsibilities will differ between teams. In an organization with web and native developers, the web developers will be responsible for setting up the apps in Appflow and performing web builds/deployments, while the native developers will be responsible for configuring the Live Updates SDK and using the Portals CLI to pull down the appropriate web bundles during build time.
+
+## Accessing the debug project
+
+We will be using the [portals-debug-app](https://github.com/ionic-team/portals-debug-app) repository for the purpose of this tutorial, however Appflow will support any web app that can be compiled with npm or yarn.
+
+Start by forking or cloning the [portals-debug-app](https://github.com/ionic-team/portals-debug-app) repository. If you choose to clone the repository, you should push a copy of it to the git provider of your choice (Appflow supports Github, Gitlab, Bitbucket, and Azure DevOps). 
+
+If you are using your own web app rather than the demo project, then you should add a `capacitor.config.json` file to the root of your web project. Capacitor is not technically needed as a web project dependency, but the configuration file will help provide some configuration options for Portals. The following configuration can be used:
+```json
+{
+  "appId": "io.ionic.starter",
+  "appName": "Portals Debug Web App",
+  "webDir": "dist"
+}
+```
+
+Our demo project already contains a `capacitor.config.ts` file which gets compiled to a json file during the build process, so adding one to the debug project is not necessary.
+
+## Creating a new app
```

Comment:
Can we change this to "Creating a new Appflow app"? Just to differentiate between a web app and an Appflow app.